packages/controller/src/resources/monitoring/system/dashboards/tenant-overview-loki.json

```diff
@@ -1,924 +1,924 @@
 {
-    "annotations": {
-      "list": [
+  "annotations": {
+    "list": [
```

Comment:
That diff is not so optimal! :)

Reply:
Sorry, I should have done whitespace changes separately.